### PR TITLE
Problem: (fix #17) some crates version is not specified in Cargo.toml

### LINKED ISSUE #17: Problem: some crates version is not specified in Cargo.toml
use the crates in crates.io instead of the one from github.com

### PR DESCRIPTION
**Pull request by linfeng-crypto**

### CODE REVIEW

File: .github/workflows/chainlib.yml [hunk 1]
```diff
@@ -62,7 +66,7 @@ jobs:
           toolchain: ${{ matrix.rust }}
           target: ${{ matrix.target }}
           override: true
-      - run: cargo build --target ${{ matrix.target }} --release
+      - run: cargo build --target ${{ matrix.target }} --release --features=${{ matrix.feature }}
```
Comment (tomtau) [inline .github/workflows/chainlib.yml:69]: are the features mutually exclusive ? what about build with `--all-features`?

File: src/ledger_app.rs [hunk 2]
```diff
@@ -0,0 +1,171 @@
+/*******************************************************************************
```
Comment (tomtau) [inline src/ledger_app.rs:1]: also update https://github.com/crypto-com/chainlib-rs/blob/main/NOTICE

**Review by tomtau — COMMENTED** [review]
looks ok

File: src/ledger_app.rs (hunk 2 above)
Comment (linfeng-crypto) [reply src/ledger_app.rs:1]: should I remove this comment?

File: .github/workflows/chainlib.yml (hunk 1 above)
Comment (linfeng-crypto) [reply .github/workflows/chainlib.yml:69]: yes, it seems ok to use `--all-features`, I'll use it.

**Pull request merged**